boot/zephyr/Kconfig

```
@@ -388,6 +388,13 @@ config RECOVERY_UART_DEV_NAME
 
 endif # MCUBOOT_SERIAL
 
+config BOOT_INTR_VEC_RELOC
+	bool "Relocate the interrupt vector to the application"
+	default n
+	depends on SW_VECTOR_RELAY || CPU_CORTEX_M_HAS_VTOR
+	help
+	  Relocate the interrupt vector to the application before it is started.
```

Add hint on why this is needed to the help message:
```suggestion
	  Relocate the interrupt vector to the application before it is started.
	  Select this option if application requires vector relocation,
	  but it doesn't relocate vector int its reset handler.
```
Select this option if application requires vector relocation, but it doesn't relocate vector int its reset handler.

---

Thank you. I have added the change.